src/libgpu/src/vulkan/vulkan_driver.h

```diff
@@ -50,11 +51,6 @@ class HashedDesc
       return mHash;
    }
 
-   operator bool() const
```

Comment:
Why is this removed here?

Reply:
It is unused method. What would be even the meaning of bool(hash) ?

Reply:
It used to be used to verify if there is actually a hash stored (the hash uint64_t[2]{0, 0}).  I think I ended up getting rid of it in exchange for checking if the last usage is non-zero.